### ISSUE TITLE
Make a Thicc Footer

### ISSUE THREAD

**Issue by Crimson-Blade** [Manipal-OSF/website#45]
![image](https://user-images.githubusercontent.com/71289117/220247630-de8bc43a-534e-4a5c-aa73-30a8079f137e.png)
Something like this. Beeg

**Comment by Crimson-Blade** [comment]
The thing is about 524px tall in this case. Just for reference

**Comment by dhzdhd** [comment]
Yeah, can include the back to top too when we add more content in the landing page
